### PR TITLE
Added some renamed stations.

### PR DESCRIPTION
**Pull request by juliuste**
Also removed "nolle" ∈ "nollendorfplatz".

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -15,13 +15,22 @@ module.exports = {
...
-	"lehrter bahnhof": 9003201, // S+U Berlin Hauptbahnhof
+	"eichkamp":        			9025423, // S Messe Süd (Berlin)
+	"witzleben":       			9024106, // S Messe Nord/ICC (Berlin)
+	"lehrter bahnhof": 			9003201, // S+U Berlin Hauptbahnhof
+	"lindenthaler allee": 		9050301, // S Mexikoplatz (Berlin)
+	"unter den linden": 		9100025, // S+U Brandenburger Tor (Berlin)
+	"papestrasse": 				9058101, // S Südkreuz (Berlin)
+	"zinnowitzer strasse": 		9100009, // U Naturkundemuseum (Berlin)
+	"seidelstrasse":			9086101, // U Otisstr. (Berlin)
+	"knie":						9023101, // U Ernst-Reuter-Platz (Berlin)
```
Comment (derhuerst) [inline index.js:31]: "knie" is a slang name right? Would you mind to move this line up?

Comment (juliuste) [inline index.js:31]: "Knie" is not a slang name, but rather the former official name of the station.
(https://de.wikipedia.org/wiki/Ernst-Reuter-Platz)

Comment (juliuste) [inline index.js]: Source: https://de.wikipedia.org/wiki/Ernst-Reuter-Platz

Comment (derhuerst) [inline index.js:31]: Crazy. Will merge.

**Pull request merged**